### ISSUE TITLE
What the status of this document?

### ISSUE THREAD

**Issue by T-bagwell** [veovera/enhanced-rtmp#6]
Is this document stoped?
Or is this document will be updated? I mean modify the FourCC for codec.

**Comment by veovera** [comment]
Yes, we will have "additional capabilities to continue to modernize RTMP".
